src/tests/multihost/ad/test_access_control.py

```diff
@@ -0,0 +1,409 @@
+
+    @staticmethod
+    def test_002_too_much_logging_from_sssd_be(multihost, create_aduser_group, backupsssdconf):
+        """
+        :title: too much logging from sssd_be bz1269018
+        :description: sssd_be is logging in the authentications in /var/log/messages
+        :id: 82dd0c1a-cf62-4e15-978f-2a6e6c4c007c
+        :steps:
+            1. Clear /var/log/messages
+            2. AD-user log in multiple times via ssh
+        :expectedresults:
+            1. /var/log/messages should be empty
+            2. /var/log/messages should not have logs related sssd_be
+        """
+        (aduser, _) = create_aduser_group
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        dom_section = f'domain/{domain_name}'
+        sssd_params = {'access_provider': 'simple',
+                       'simple_allow_users': f'{aduser}@{domain_name}'}
+        tools.sssd_conf(dom_section, sssd_params, action='add')
+        tools.clear_sssd_cache()
+        multihost.client[0].run_command('echo > /var/log/messages', raiseonerr=False)
+        client_hostname = multihost.client[0].sys_hostname
+        for i in range(4):
+            locals()[f'cl{i}'] = pexpect_ssh(client_hostname, f'{aduser}@{domain_name}', 'Secret123', debug=False)
+            client = locals()[f'cl{i}']
+            try:
+                client.login()
+            except SSHLoginException:
+                pytest.fail(f'{aduser} failed to login')
+            else:
+                client.logout()
+        log_str = multihost.client[0].get_file_contents('/var/log/messages').decode('utf-8')
+        patt = re.compile(r'sssd_be')
+        assert not patt.findall(log_str), 'The /var/log/messages is populated with sssd_be related logs'
+
+    @staticmethod
+    def test_003_simple_allow_user_to_dollar_symbol(multihost, create_aduser_group, backupsssdconf):
+        """
+        :title: Set simple_allow_user to dollar symbol
+        :description: Setting simple_allow_user to '$' to deny all user log in
+        :id: 3317cb9f-fa25-40b4-835b-d8c5da9cda5b
+        :steps:
+            1. Set simple_allow_user to $ symbol in sssd.conf and restart sssd
+            2. Log in as AD user
+        :expectedresults:
+            1. Changes should reflected in restarted sssd
+            2. Any AD-User log in should be denied
+        """
+        tools = sssdTools(multihost.client[0])
+        (aduser, _) = create_aduser_group
+        domain_name = tools.get_domain_section_name()
+        dom_section = f'domain/{domain_name}'
+        sssd_params = {'access_provider': 'simple',
+                       'simple_allow_users': '$'}
+        tools.sssd_conf(dom_section, sssd_params, action='add')
+        tools.clear_sssd_cache()
+        _, stdout = ssh_login(multihost, f'{aduser}@{domain_name}')
+        assert stdout, 'Access denied log is absent in /var/log/secure'
```

Comment:
Why is there no 'Access denied' in /var/log/secure, shouldn't there be one in case the use is rejected?

Why are you not using the first output, the `status`, for the check?

Reply:
> > Why is there no 'Access denied' in /var/log/secure, shouldn't there be one in case the use is rejected?
>  The 'Access denied' is indeed being asserted in /var/log/secure.  The message after the comma ',' in the test will be printed only when the test fails.

Hi,

thanks for the explanation. But I think in this case the `assert` will never fail because if I understand it correctly `ssh_login()` will always set `stdout` and `assert stdout` will only fail if `stdout` is not set.

bye,
Sumit

> > Why are you not using the first output, the `status`, for the check?
>  It could be used for checking. The original bash tests have relied on the assertion of 'denied' log presence. The 'status' could be used confirm that if second output is as expected

Reply:
Added the check for the first output. and log presence.